frontend/src/components/Graph.jsx

```diff
@@ -9,19 +9,19 @@ function Graph({ rawDocs, displayedDocs }) {
 
   useEffect(() => {
     let docs = rawDocs || [];
-    const types = Array.from(new Set(docs
-      .flatMap(d => d[2])
-      .filter(x => x !== undefined)
-      .map(x => x.verb)
-    ));
-    const typeColorScale = scaleOrdinal(types, schemeCategory10);
+    const types = {
+      adapts: 'is adapted from',
+      refersTo: 'is referred by',
+      includes: 'is included in',
+    };
+    const typeColorScale = scaleOrdinal(Object.values(types), schemeCategory10);
     const nodes = docs.map(([id, title]) => ({id, title}));
     const links = docs.flatMap(d =>(d[2] || [])
       .filter(l => displayedDocs.includes(l.object.split('#')[0]))
       .map(l => ({
-        source: d[0],
-        type: l.verb,
-        target: l.object.split('#')[0]
+        source: l.object.split('#')[0],
+        target: d[0],
+        type: types[l.verb]

```

Comment:
What if the verb is unknown? 

You could generate something like `{verb} (inverse)`...

Reply:
Undefined labels are now OK. Thank you @CocoChart.
<img width="617" alt="Capture d’écran 2025-04-29 à 09 57 56" src="https://github.com/user-attachments/assets/01140873-aa31-4950-bdc6-ae7791a90700" />

However, how comes that there are no arrows anymore? 

<img width="617" alt="Capture d’écran 2025-04-29 à 09 57 42" src="https://github.com/user-attachments/assets/9097e084-37ad-44e9-bc07-c4da11574ee9" />

Reply:
It's weird, the arrows work fine on my local version . The problem might come from adding new types that are not in the array. I've just pushed a new version that may fix this